fea-rs/src/parse/context.rs

```diff
@@ -431,9 +428,9 @@ mod tests {
         )
         .unwrap();
 
-        let a_id = parse.sources.id_for_path("file_a").unwrap();
-        let b_id = parse.sources.id_for_path("b").unwrap();
-        let c_id = parse.sources.id_for_path("c").unwrap();
+        let a_id = parse.sources.id_for_path(Path::new("file_a")).unwrap();
```

Comment:
I'd leave the signature of `id_for_path` to take `impl AsRef<Path>`, and then we wouldn't need any of these changes (since `str` impls `AsRef<Path>`).

Reply:
Made it so